src/core/thread/fiber.d

```diff
@@ -954,9 +954,11 @@ private:
             m_ctxt.tstack = pbase;
             m_size = sz;
         }
-        else
+        else version (Posix)
         {
-            version (Posix) import core.sys.posix.sys.mman; // mmap, MAP_ANON
+            import core.sys.posix.sys.mman; // mmap, MAP_ANON
```

Comment:
This imported module will always define `mmap` or else compilation will fail with  `static assert(false, "Unsupported platform")`. So there seems to be no way the `malloc` branch can ever be reached.

https://github.com/dlang/druntime/blob/31c9e38d4b5d028a7cc42a09203734ba6609c959/src/core/sys/posix/sys/mman.d#L261-L326

Reply:
Yes, I brought this fix by accident and did `git push -f`. :-(

I think we just need to get rid of these `__traits( compiles, mmap )` at whole file replacing them by `version(POSIX_MMAP)`, etc.